<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Power Theory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="plotGroupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Plot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="PlotWidget" name="plotGraphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>561</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>101</width>
       <height>228</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>120</number>
      </property>
      <item>
       <widget class="QComboBox" name="plotTypeComboBox">
        <item>
         <property name="text">
          <string>Voltage</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Current</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="line1CheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Line 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="line2CheckBox">
          <property name="text">
           <string>Line 2</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="line3CheckBox">
          <property name="text">
           <string>Line 3</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="zoomInButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomOutButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>330</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom Out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="historyButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>History</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="inputGroup1GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>190</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Group 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="inputGroup1GroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>190</width>
      <height>185</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Group 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>plotTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onPlotTypeComboBoxSelect(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>historyButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onHistoryButtonClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomInButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomOutButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line1CheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onLine1CheckBoxClickedHistoryPlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line2CheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onLine2CheckBoxClickedHistoryPlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line3CheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onLine3CheckBoxClickedHistoryPlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPlotTypeComboBoxSelect(int)</slot>
  <slot>onHistoryButtonClicked(bool)</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>onLine1CheckBoxClickedHistoryPlot(bool)</slot>
  <slot>onLine2CheckBoxClickedHistoryPlot(bool)</slot>
  <slot>onLine3CheckBoxClickedHistoryPlot(bool)</slot>
 </slots>
</ui>
